Analysis/DEEP2_R23_O32/Archives/equivalent_width_plots.py

```diff
@@ -0,0 +1,66 @@
+
+"""
+PLOTTING RESULTS FROM ANALYSIS
+Some were written before MSC and are now in MSC
+
+Functions:
+-Equivalent Width for R23
+-Equivalent Width for O32
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.io import ascii as asc
+from matplotlib.backends.backend_pdf import PdfPages
+from os.path import join
+
+from Zcalbase_gal.Analysis.DEEP2_R23_O32 import zoom_and_gauss_general
+from Metallicity_Stack_Commons.Metallicity_Stack_Commons import lambda0, line_type, line_name
+
+fitspath_ini = '/Users/reagenleimbach/Desktop/Zcalbase_gal/'
+
+
+def ew_plot_r23(fitspath, asc_table):
+    """
+    Purpose
+    Plots equivalent width verse R23 and saves to a pdf file.
+    """
```

Comment:
Note that if this is a one-liner, you can always do:
```
"""Plots equivalent width verse R23 and saves to a pdf file."""
```

Since this is part of archives, it's not crucial.

Reply:
Fixed